srcpkgs/xbps-triggers/files/dkms

```diff
@@ -30,8 +30,8 @@ remove_modules() {
 			if $(echo "$line" | egrep -vq '(added|built|installed)'); then
 				shift 2; continue
 			fi
-			_modver=$(echo "$line"|sed "s/$1,[[:blank:]]\([^,]*\)[,:].*/\1/;t;d")
-			_kver=$(echo "$line"|awk '{print $3}'|sed "s/\(.*\),$/\1/")
+			_modver=$(echo "$line"|sed "s/[^/]*\/\([^\,]*\).*/\1/")
+			_kver=$(echo "$line"|awk '{print $2}'|sed "s/\(.*\),$/\1/")
```

Comment:
Please explain the changes motivating this change in the commit message.

Reply:
I am afraid I do not know English well enough to write such a comment.
):

Reply:
I think you could link the upstream commits mentioned in the PR.

And including an example of the old output vs new output.